### PR TITLE
Make it possible to override content-type and content-encoding from CLI arguments

### LINKED ISSUE #264: How to add custom metadata e.g "cache-control"?
Official AWS CLI does support adding metadata such as `--content-type` and `--cache-control`. 

```
$ aws s3 sync ./mydir/ s3://example-bucket/mydir/ \
--content-type 'application/rss+xml; charset=UTF-8' \
--cache-control 'max-age=86400'
```

The reason I need custom metadata is because I use S3 as static website hosting which sits behind Cloudfront. Setting correct origin headers such as `cache-control` and `content-type` for CDN/Proxy is important to my case.

How do I add custom metadata when uploading object using s5cmd? It seems I did not find the options.

### PR DESCRIPTION
**Pull request by nattofriends**
This PR lets users override Content-Type and Content-Encoding headers, to close #264. #318 is a similar PR which added the --cache-control and --expires options.

### CODE REVIEW

**Comment by nattofriends** [comment]
Oops, didn't notice #466 exists too

File: command/cp.go [hunk 1]
```diff
@@ -557,6 +569,13 @@ func (c Copy) doUpload(ctx context.Context, srcurl *url.URL, dsturl *url.URL) er
 		SetCacheControl(c.cacheControl).
 		SetExpires(c.expires)
 
+	if c.contentType != "" {
+		metadata.SetContentType(c.contentType)
```
Comment (sonmezonur) [inline command/cp.go:573]: If `content-type` is given, there is no need to calculate/guess content-type. Could you please remove line [564](https://github.com/peak/s5cmd/blob/330c26cac176bbaad5690ed5c239b04a17b0f345/command/cp.go#L564) and call `guessContentType` only if content-type flag is empty.

**Review by aykutfarsak — APPROVED** [review]

**Comment by sonmezonur** [comment]
@nattofriends Thanks for your contribution. Could you please update CHANGELOG?

**Comment by sonmezonur** [comment]
Also, adding examples to cp help text would be useful.

**Review by sonmezonur — APPROVED** [review]

**Comment by sonmezonur** [comment]
🚀

**Pull request merged**